app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: ScihistDigicoll::Env.lookup!(:no_reply_email_address)
```

Comment:
Nice catch for consistency, but =this will raise on loading the class if no Env `no_reply_email_address` is provided. And our class doesn't provide a default in dev/test. So. 

(I'm actually surprised and a bit alarmed that tests pass with this, not sure if I don't understand something, or if tests just don't cause this class to be loaded). 

Maybe change to `lookup` without the `!`, which will just return nil instead of raising. Or, backwards compatibly, `(ScihistDigicoll::Env.lookup(:no_reply_email_address) || "from@example.com")

Reply:
Good call. Will use the latter.